dev/ProgressRing/ProgressRing.cpp

```diff
@@ -341,7 +341,8 @@ void ProgressRing::UpdateStates()
             // Swap player source to determinate.
             SetAnimatedVisualPlayerSource();
             UpdateLottieProgress();
-        }   
+        }
+        winrt::AutomationProperties::SetAccessibilityView(*this, winrt::AccessibilityView::Content);
```

Comment:
Can we add a quick test ?

Reply:
Added an API test for this now (and the missing API tests project for the ProgressRing)